### DISCUSSION TITLE
Welcome to obsidian-apple-books-highlights-plugin Discussions!

### DISCUSSION THREAD

**Discussion by bandantonio** [bandantonio/obsidian-apple-books-highlights-plugin#31] [Announcements]
## 👋 Hi and welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ElijahLewes** [comment]
Hey! Quick question - I'm getting a "Folder already exists" error when the plug-in loads on launch, but haven't been able to id the path it's referring to. Any ideas on how to fix this?

Thanks!

**Comment by bandantonio** [comment]
Hi @ElijahLewes 
The plugin itself doesn't return this error, it seems like the error is on the Obsidian level.
Did you encounter the error right after the first installation of the plugin? Provide more context, please.

**Comment by 5Net** [comment]
How can I obtain the chapter names of a book? Or what kind of books can have their chapter names retrieved?When I'm searching my notes, I want to see which chapter the highlighted content belongs to

**Reply by bandantonio** [reply]
Hi @5Net 
Thanks for your patience.
To obtain chapter names, you can use the following syntax inside the `annotations` loop:

```hbs
{{#each annotations}}
- 📖 Chapter:: {{#if chapter}}{{{chapter}}}{{else}}N/A{{/if}}
{{/each}}
```

However, chapters may return `N/A`: it is expected behavior. I provided more details about that in [my old reply](https://github.com/bandantonio/obsidian-apple-books-highlights-plugin/discussions/32#discussioncomment-10152546)
